src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/RemoteAppServerExtensions.cs

```diff
@@ -25,29 +25,10 @@ public static ISystemWebAdapterBuilder AddRemoteAppServer(this ISystemWebAdapter
         }
 
         builder.Services.AddOptions<RemoteAppServerOptions>()
+            .Configure(configure)
             .ValidateDataAnnotations();
 
-        configure(new Builder(builder.Services));
-
-        return builder;
-    }
-
-    public static ISystemWebAdapterRemoteServerAppBuilder Configure(this ISystemWebAdapterRemoteServerAppBuilder builder, Action<RemoteAppServerOptions> configure)
-    {
-        if (builder is null)
-        {
-            throw new ArgumentNullException(nameof(builder));
-        }
-
-        if (configure is null)
```

Comment:
Again, probably want to keep the null check

Reply:
same - the method was removed. the null check is still in the existing method